docs/data/routes/docs/nextjs/data-fetching/data-services/en.md

```diff
@@ -3,4 +3,38 @@ name: data-services
 routeTemplate: ./data/component-templates/article.yml
 title: Data services used by Next.js with JSS
 ---
-# Data services used by Next.js with JSS
\ No newline at end of file
+# Data services used by Next.js with JSS
```

Comment:
Worth some sort of note here that if you have data needed by every page, you can di additional data fetching in the props factory.

Reply:
@nickwesselman agree, mentioned about it